geth/testing/testing.go

```diff
@@ -110,15 +113,23 @@ func FirstBlockHash(require *assertions.Assertions, nodeManager common.NodeManag
 	require.Equal(expectedHash, firstBlock.Hash.Hex())
 }
 
+// MakeTestNodeConfig returns the default params.NodeConfig for testing.
 func MakeTestNodeConfig(networkID int) (*params.NodeConfig, error) {
+	testDir := filepath.Join(TestDataDir, TestNetworkNames[networkID])
+
+	if runtime.GOOS == "windows" {
+		testDir = filepath.ToSlash(testDir)
+	}
```

Comment:
I understand but I believe `filepath.Join` already does that for such kind of paths on Windows. No?
https://golang.org/pkg/path/filepath/#Join